_episodes_rmd/05-data-visualization.Rmd

```diff
@@ -142,129 +133,163 @@ Building plots with **`ggplot2`** is typically an iterative process. We start by
 defining the dataset we'll use, lay out the axes, and choose a geom:
 
 ```{r create-ggplot-object, purl=FALSE}
-ggplot(data = metadata, aes(x = sample, y = generation)) +
-geom_point()
+ggplot(data = variants, aes(x = POS, y = DP)) + 
+  geom_point()
 ```
 
-Alternatively, we can make our data points slightly transparent. We
-do this by adding an aesthetic within our `geom_point()` function 
-call. The name of the aesthetic we want is "alpha". 
+Then, we start modifying this plot to extract more information from it. For
+instance, we can add transparency (`alpha`) to avoid overplotting:
 
-```{r alpha, purl=FALSE}
-ggplot(data = variants, aes(x = POS, y = sample_id))+
-geom_point(alpha = 0.1)
+```{r adding-transparency, purl=FALSE}
+ggplot(data = variants, aes(x = POS, y = DP)) +
+    geom_point(alpha = 0.5)
 ```
 
 We can also add colors for all the points:
 
 ```{r adding-colors, purl=FALSE}
-ggplot(data = variants, aes(x = POS, y = sample_id))+
-geom_point(alpha = 0.1, color = "blue")
+
+ggplot(data = variants, aes(x = POS, y = DP)) +
+  geom_point(alpha = 0.5, color = "blue")
 ```
 
-We could also color code each of our points according to the number of
-reads (after filtering) that support that variant. This information
-is in the `DP` column: 
+Or to color each species in the plot differently, you could use a vector as an input to the argument **color**. **`ggplot2`** will provide a different color corresponding to different values in the vector. Here is an example where we color with **`sample_id`**:
+
+```{r color-by-sample-1, purl=FALSE}
+ggplot(data = variants, aes(x = POS, y = DP, color = sample_id)) + 
+  geom_point(alpha = 0.5)
+```
 
+Notice that we can change the geom layer and colors will be still determined by **`sample_id`**
 
-```{r color-by-DP, purl=FALSE}
-ggplot(data = variants, aes(x = POS, y = sample_id))+
-geom_point(aes(color = DP))
+```{r color-by-sample-2, purl=FALSE}
+ggplot(data = variants, aes(x = POS, y = DP, color = sample_id)) + 
+  geom_jitter(alpha = 0.5)
+```
+
+To make our plot more readable, we can add axis labels:
+
+```{r add-axis-labels, purl=FALSE}
+ggplot(data = variants, aes(x = POS, y = DP, color = sample_id)) + 
+  geom_jitter(alpha = 0.5) + labs(x = "Base Pair Position", y = "Read Depth (DP)")
```

Comment:
I might move the `labs()` call to its own line, and also break the x and y labels to their own lines, to aid in readability.